types/parse/index.d.ts

```diff
@@ -457,25 +457,9 @@ declare global {
             remove: this["add"];
             removeAll: this["addAll"];
             revert(...keys: Array<Extract<keyof (T & CommonAttributes), string>>): void;
-            save<K extends Extract<keyof T, string>>(
-                attrs?:
-                    | (((x: T) => void) extends (x: Attributes) => void
```

Comment:
This is true:
```ts
((x: T) => void) extends (x: Attributes) => void
```
only when `T` has no optional key.

Reply:
To clarify here: the only reason why this case _should_ error (in my opinion) is because the attributes had been typed without allowing `undefined`. For the case where the attributes ARE typed either as `Partial` or with properties typed as `undefined`, I'm fine allowing those to be set as `undefined`.

My problem was just with the `Partial<T>`, but I think `Pick<T, K> | T` seems to work great.